www/js/controllers/map-controller.js

```diff
@@ -34,33 +35,28 @@ angular.module('pvta.controllers').controller('MapController', function($scope,
 
   // Almost the same as immediately above.
   // Query Cordova for current location first.
+  // Also fits the bounds to include everything we've added/removed from them
   var currentLocation = $cordovaGeolocation.getCurrentPosition(options).then(function(position){
     var myLocation = new google.maps.LatLng(position.coords.latitude, position.coords.longitude);
     bounds.extend(myLocation);
-    addMapListener(placeDesiredMarker(myLocation), 'You are here!');
+    $scope.map.fitBounds(bounds);
+    addMapListener(placeDesiredMarker(myLocation, 'http://www.google.com/mapfiles/kml/paddle/red-circle.png'), 'You are here!');
   }, function(error){});
 
 
   // Takes a google.maps.LatLng object and places a marker
   // on the map in the requested spot.
   // Returns a reference to said marker.
-  function placeDesiredMarker(location){
+  function placeDesiredMarker(location, icon){
     var neededMarker = new google.maps.Marker({
         map: $scope.map,
-        icon: 'http://www.google.com/mapfiles/kml/paddle/go.png',
+        icon: icon,
         animation: google.maps.Animation.DROP,
         position: location
       });
     return neededMarker;
   };
 
-  // When the map is ready, resize it to
-  // the bounds that we have been setting each
-  // time we add something to it.
-  google.maps.event.addListenerOnce($scope.map, 'idle', function(){
```

Comment:
Well because of the location of this code (it's not within a loop or conditional), it will always show your current location on the map, so it will always resize the bounds.